src/modules/billing/plugin.js

```diff
@@ -0,0 +1,8 @@
+const populateBillingBatch = require('./jobs/populate-billing-batch');
+
+module.exports = {
```

Comment:
I wonder if this plugin should call `registerSubscribers` from the deleted `jobs/init` file.

With all the publishing and subscribing that we will need to handle for billing there may end up being quite a lot of logic required which might not obviosuly live in the plugin.